shared/src/business/entities/cases/Case.ts

```diff
@@ -134,7 +133,7 @@ export class Case extends JoiValidationEntity {
   public useSameAsPrimary?: boolean;
   public initialDocketNumberSuffix?: string;
   public noticeOfTrialDate?: string;
-  public docketNumberWithSuffix?: string;
+  public docketNumberWithSuffix!: string;
```

Comment:
We always set this. See constructor. The `!` is because we set it via constructor -> `assignFieldsForAllUsers`, not directly in the constructor itself, and TypeScript complains that we don't set it directly in the constructor.